drivers/i2c/i2c_gpio.c

```diff
@@ -1,19 +1,23 @@
 /*
+ * Copyright (c) 2020 Vestas Wind Systems A/S
  * Copyright (c) 2017 Linaro Ltd.
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#define DT_DRV_COMPAT gpio_i2c
+
 /**
  * @file
  * @brief Driver for software driven I2C using GPIO lines
  *
- * This driver implements an I2C interface by driving two GPIO lines under
- * software control. The GPIO pins used must be preconfigured to to a suitable
- * state, i.e. the SDA pin as open-colector/open-drain with a pull-up resistor
- * (possibly as an external component attached to the pin). When the SDA pin
- * is read it must return the state of the physical hardware line, not just the
- * last state written to it for output.
+ * This driver implements an I2C interface by driving two GPIO lines
+ * under software control. The GPIO pins used must be configured
+ * through devicetree and pinmux with suitable flags, i.e. the SDA pin
+ * as open-colector/open-drain with a pull-up resistor (possibly as an
```

Comment:
I'd rather see major reformatting like this done in a separate commit.  If there's any actual content change here it's not obvious.

Reply:
@pabigot Good point. I have split the reformatting from the content change in two, separate commits.